test/book_test.rb

```diff
@@ -0,0 +1,18 @@
+require "minitest/autorun"
+require "minitest/pride"
+require "./lib/book"
+
+class BookTest < Minitest::Test
+  def test_it_has_attributes
+    book = Book.new({author_first_name: "Harper", author_last_name: "Lee", title: "To Kill a Mockingbird", publication_date: "July 11, 1960"})
+    
+    assert_equal "To Kill a Mockingbird", book.title
+    assert_equal "Harper Lee", book.author
+  end
+
+  def test_book_publication_year
+    book = Book.new({author_first_name: "Harper", author_last_name: "Lee", title: "To Kill a Mockingbird", publication_date: "July 11, 1960"})
+
+    assert_equal "1960", book.publication_year
+  end
```

Comment:
Also, if you add in other attributes to your Book class, there should be tests for it.